waiter/src/waiter/scheduler.clj

```diff
@@ -37,6 +37,11 @@
            (java.util.concurrent TimeoutException)
            (org.joda.time DateTime)))
 
+(defmacro log
+  "Log Scheduler-specific messages."
+  [& args]
+  `(log/log "Scheduler" :debug nil (print-str ~@args)))
```

Comment:
Query: does this include the CID in the logs?

Reply:
To clarify, I updated the logging format in the log4j properties file to include the CID.